fili-core/src/main/java/com/yahoo/bard/webservice/web/JobsApiRequest.java

```diff
@@ -95,34 +81,6 @@ public JobsApiRequest(
     }
 
     /**
-     * Get a PreResponse associated with a given ticket if a timeout does not occur in 'asyncAfter' milliseconds.
-     * If the notification that a job has been stored in the PreResponseStore is received from the
-     * BroadcastChannel before the timeout, fetch the PreResponse for the specified ticket from the PreResponseStore.
-     * In case of a timeout, return an empty Observable.
-     * <p>
-     * BroadCastChannel is a hot observable i.e. it emits notification irrespective of whether it has any subscribers.
-     * We use the replay operator so that the preResponseObservable upon connection, will begin collecting values.
-     * Once a new observer subscribes to the observable, it will have all the collected values replayed to it.
-     *
-     * @param ticket  The ticket for which the PreResponse needs to be retrieved.
-     *
-     * @return An Observable wrapping a PreResponse or an empty Observable in case a timeout occurs.
-     */
-    public Observable<PreResponse> handleBroadcastChannelNotification(@NotNull String ticket) {
-        ConnectableObservable<PreResponse> broadCastChannelPreResponseObservable = broadcastChannel.getNotifications()
-                .filter(ticket::equals)
-                .timeout(asyncAfter, TimeUnit.MILLISECONDS, Observable.empty())
-                .flatMap(preResponseStore::get)
-                .take(1)
-                .replay(1);
-
-        //connect to the broadCastChannelPreResponseObservable so that it begins collecting values
-        broadCastChannelPreResponseObservable.connect();
-
-        return broadCastChannelPreResponseObservable;
-    }
```

Comment:
This has been moved into the method [JobsServlet::getResults](https://github.com/yahoo/fili/pull/39/files#diff-97b8f8f634a71f2ff451f07dbecd29dcR363)